README.md

```diff
@@ -1,13 +1,15 @@
 # ST 429-20 1ED (1st edition)
 Public CD of [SMPTE ST 429-20 1ED](21DC-CD-ST-429-20-202X-1ED.pdf)
 
+This document is made available for a public review period ending no earlier than `2022-01-23`, and no later than `2022-07-23`.
+
 Please consult [CONTRIBUTING.md](./CONTRIBUTING.md), [LICENSE.md](./LICENSE.md) and [PATENTS.md](./PATENTS.md) for important notices.
 
 ## Project details
 
 This document specifies constraints on SMPTE ST 377-1 for use in D-Cinema applications. It is intended to replace references to SMPTE ST 377:2004, allowing D-Cinema to benefit from the improvements of ST 377-1:2019 while retaining compatibility with ST 377:2004
 
 [GitHub issues](https://github.com/SMPTE/st429-20/issues) are preferred for discussion of this specification. Alternatively, comments can be sent to [21dc-chair@smpte.org](mailto:21dc-chair@smpte.org).
-```
 
+```
 See AG-22, Annex D for additional requirements specific to Public Committee Drafts.
```

Comment:
```suggestion
```

Reply:
updated in e1a0ac4, sorry there was a GH caching issue apparently.